CONTRIBUTING.md

```diff
@@ -38,6 +38,16 @@ application.
 When you make further changes to the extension, quit the "Extension Development
 Host" and run `F5` (or `Run > Start debugging`) again.
 
+## Publishing and testing locally
+
+To ensure that everything is working as expected after changes it's best to
+publish a vsix artifact locally. You can do this by `yarn build` which will result in
+the metals-<version>.vsix being built and placed in the root of your directory.
+After the artifact has successfully build you can install the extension by going to
+the extension menu in Visual Studio Code and under "More Actions..." choose to
```

Comment:
There’s also a way to do this from the command line. I’m from mobile right now but it should be easy to find and it may make sense to document it here

Reply:
Sure, I added that in as well. 👍